content/asset-tracking/index.textile

```diff
@@ -0,0 +1,157 @@
+---
+title: Asset Tracking
+section: asset-tracking
+index: 0
+meta_description: "The Ably asset tracking solution provides a way to track multiple assets in realtime."
+meta_keywords: "Ably, Asset Tracking SDK, Mapbox, Mapbox Navigation SDK, Mapbox location enhancement, Ably realtime"
+languages:
+  - javascript
+  - android
+  - objc
+  - java
+  - swift
+jump_to:
+  Help with:
+    - Key participants#key-participants
+    - Resolution policy#resolution-policy
+    - See also#see-also
+---
+
+The Ably asset tracking solution provides a way to track multiple assets in realtime. An asset is any object that moves geographically, and whose position needs to be monitored. The asset tracking solution is powered by the Ably realtime network, the Ably Asset Tracking SDKs, and the Mapbox Navigation SDK with location enhancement.
+
+The Ably asset tracking solution provides two SDKs:
+
+* Publisher SDK - for embedding in apps on the asset to be tracked.
+* Subscriber SDK - for embedding in apps that want to observe the asset being tracked.
+
+As Ably is used as the underlying transport, you have direct access to your data and can use Ably integrations for a wide range of applications, in addition to direct realtime subscriptions. Examples include: 
+
+* Passing data to a third-party service.
+* Persistence of data to a database for later retrieval.
+
+As the asset being tracked moves, it publishes its position through the app to an Ably channel. Any subscribers to the channel can then receive positional information, which can be displayed on a detailed map.
+
+The following diagram provides an overview of an asset tracking use case:
+
+<a href="/images/screenshots/asset-tracking/overview.png" target="_blank">
+  <img src="/images/screenshots/asset-tracking/overview.png" style="width: 100%" alt="Overview">
+</a>
+
+In a typical asset tracking scenario the key participants are:
+
+- asset (trackable) := Item that the customer wants to track such as driver, vehicle, container, and so on. This also known as a @Trackable@ in the SDKs.
+- driver := Person driving the asset physically, or entity controlling the asset programatically.
+- customer := Entity that uses the Ably Asset Tracking SDKs to build their product to solve the asset tracking problem for their users.
+- end user := Person that uses the product built by the customer and uses the asset tracking capabilities.
+
+Each asset being tracked must have a @trackableId@, and this must be globally unique within an Ably asset tracking account.
+
+The Ably Asset Tracking SDKs also allow for various battery optimization scenarios, so frequency of updates can be modified based on various parameters such as distance to destination, subscriber status, map zoom level, and existing battery levels.
```

Comment:
"subscriber preferences" instead of "subscriber status"

Reply:
Should it be just "battery levels" instead of "existing battery levels"?

Reply:
> Should it be just "battery levels" instead of "existing battery levels"?

I've changed to "current battery levels". The idea being if the battery level is low, resolution can be modified.